docs: add starlight-page-context-action to community plugins (#4055)
withastro/starlight: docs/src/content/docs/resources/plugins.mdx

		title="starlight-biel"
		description="Add an Ask AI chatbot that answers from your content, powered by Biel.ai."
	/>
	<LinkCard
		href="https://github.com/babblebey/starlight-page-context-action"
		title="starlight-page-context-action"

> 		description="A starlight plugin that adds page action buttons to your documentation, making utility actions like copy/view page as markdown, open in AI chat, and scroll to top always accessible."

Reviewer: I might suggest trimming this down slightly as the description is getting a bit big on the button here:

<img width="398" height="223" alt="link from the deploy preview showing the description wrapping across five lines" src="https://github.com/user-attachments/assets/f5dc150b-4de5-4c12-bfa9-f24b286f36dc" />

Maybe something like this?

```suggestion
		description="Make utility actions like copy/view page as markdown, open in AI chat, and scroll to top always accessible in your docs."
```
